README.md

```diff
@@ -52,6 +52,8 @@ To update the plugin submodule paste the following command in the terminal from
 ## Quick Start
 
 A demo map is included in the plugin for demonstrating how the **WebView** opens at runtime. It is located in the `RpmWebView\Content\Maps` folder.
+The webview will not run properly until the **Subdomain** of your application is set in the project settings. You can find them in **Ready Player Me Studio** website.
+The **Subdomain** should belong to the same application otherwise the webview will fail.
```

Comment:
I don't know if this sentence (the subdomain should belong to the same application....) is necessary. I think it was copy and paste from the AppID + subdomain stuff from avatar creator?